Optimize (CQA) - Mng content - Managing CVs (#4897)
theforeman/foreman-documentation: guides/common/modules/con_composite-content-views-overview.adoc

= Composite content views overview

[role="_abstract"]

> A composite content view merges repositories from several content views into one host-facing repository.

Reviewer: Similar to what I wrote in https://github.com/theforeman/foreman-documentation/pull/4895#discussion_r3347029131.

Author: Does this work?

```suggestion
A composite content view combines repositories from several content views for hosts to consume content.
```

> Use it when hosts need content from multiple views, such as an operating system and an application, managed separately.

Reviewer: ```suggestion
Use it when hosts need content from multiple content views, such as an operating system and an application, managed separately.
```
